src/core_modules/capture-core/hooks/useProgram/useProgram.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { useMemo } from 'react';
+import { getProgramFromProgramIdThrowIfNotFound, TrackerProgram } from '../../metaData';
+import { programTypes } from './programTypes.const';
+
+export const useProgram = (programId: string) => useMemo(() => {
+    const program = getProgramFromProgramIdThrowIfNotFound(programId);
+    return {
+        program,
+        programType: program instanceof TrackerProgram ? programTypes.TRACKER_PROGRAM : programTypes.EVENT_PROGRAM,
+    };
+}, [programId]);
```

Comment:
I have created a very similar hook lately but isnt yet merged. Perhaps we could align on this

Reply:
ok, good. Let's align when you merge the code.

Reply:
Yep, you will see it on some of the open PRs I have. :)